src/components/Graph.react.js

```diff
@@ -90,24 +92,58 @@ const RealPlotlyGraph = asyncDecorator(PlotlyGraph, () =>
     LazyLoader.plotly().then(LazyLoader.graph)
 );
 
-class ControlledPlotlyGraph extends PureComponent {
-    render() {
-        return (
-            <Suspense fallback={null}>
-                <RealPlotlyGraph {...this.props} />
-            </Suspense>
-        );
-    }
-}
+const ControlledPlotlyGraph = memo(props => {
+    const {className, id} = props;
+
+    const extendedClassName = className
+        ? 'dash-graph ' + className
+        : 'dash-graph';
+
+    return (
+        <Suspense
+            fallback={
+                <div
+                    id={id}
+                    key={id}
+                    className={`${extendedClassName} dash-graph--pending`}
+                />
+            }
+        >
+            <RealPlotlyGraph {...props} className={extendedClassName} />
+        </Suspense>
+    );
+});
+
+ControlledPlotlyGraph.propTypes = PropTypes.any;
```

Comment:
@Marc-Andre-Rivet I'm seeing a funny warning message in debug mode:
```
The prop `isRequired` is marked as required in `<<anonymous>>`,
but its value is `undefined`
```

That I think is coming from here... Should this just be a copy of `PlotlyGraph.propTypes` perhaps?